build/bin/build-server-rpm

```diff
@@ -231,6 +231,17 @@ build() {
                 cp -r /home/builder/clickhouse-rpm/rpmbuild/RPMS/* /home/builder/rpm/RPMS/${spec_name}-${rpm_version}
                 ls -la /home/builder/clickhouse-rpm/rpmbuild/SOURCES
                 cp -r /home/builder/clickhouse-rpm/rpmbuild/SOURCES/* /home/builder/rpm/SOURCES
+            elif [ \"${spec_name}\" == \"dbaas-tools\" ]; then
+                export PATH=\${PATH/\"/usr/local/bin:\"/}:/usr/local/bin
+
+                echo \"====> RPM build dependencies\"
+                sudo yum install -y https://github.com/goreleaser/goreleaser/releases/download/v0.142.0/goreleaser_amd64.rpm
```

Comment:
That's the only line that is different between that block and `else` block below, right? So can we move it to the spec?

Reply:
Yes, you're right, but there's no way to add a URL of package into `BuildRequires:` in RPM spec file

We can add these into baseimage which is used to build RPMs inside PMM project

Reply:
I'm not opposing to adding it to the base image, but: why we can't run that command in `%build` section? Or install a binary release instead of RPM?

Reply:
Because `%build` section is for build steps, not for installing dependencies, RPM handles it by `BuildRequires:`

Reply:
But _this_ place is for nothing at all :)